ubiconfig/_impl/loaders/base.py

```diff
@@ -26,3 +29,25 @@ def load_all(self, recursive=False):
         or remote
         """
         raise NotImplementedError()
+
+    def load_by_cs_label(self, cs_label):
+        """
+        Load all the config files from repo and return one containing config for
+        the provided content set label.
```

Comment:
I don't think we should expose the implementation of the method here. Rather, would be better to comment only on the outcome. It returns an ubiconfig matching the input or output content set label. Raises if there is no config for that label.